nltk/tokenize/stanford_segmenter.py

```diff
@@ -64,15 +63,9 @@ def __init__(self,
                 env_vars=('STANFORD_SEGMENTER',),
                 searchpath=(), url=_stanford_url,
                 verbose=verbose)
-        slf4j = find_jar(
```

Comment:
I have finished the tests locally. The reason for using 'PLACEHOLDER' as default value is that  if path_to_jar is not None (like 'slf4j-api.jar'), the find_jar_iter in internals.py would execute:

    if path_to_jar is not None:
        if os.path.isfile(path_to_jar):
            yielded = True
            yield path_to_jar
        else:
            raise LookupError('Could not find %s jar file at %s' %
                            (name_pattern, path_to_jar))

So in default situation, we need to set path_to_jar to None.